### PR TITLE
School logo

### PR DESCRIPTION
**Pull request by raoaliahmad**

### CODE REVIEW

File: web/src/components/Layout/index.js [hunk 1]
```diff
@@ -17,10 +17,18 @@ const Header = ({user, history}) => <div className="header">
...
+					<img className="header-logo" src={logo} alt="No Logo"/>
+				</div>
+				
+				<div>
+					<div className="title">{settings.schoolName}</div>
+					<div className="address">{settings.schoolAddress}</div>
+					<div className="phone-number">{settings.schoolPhoneNumber}</div>
+				</div>
+			</div>
 		</div>
```
Comment (taimur38) [inline web/src/components/Layout/index.js]: why do we have so many divs all of a sudden

File: web/src/components/Layout/style.css [hunk 2]
```diff
@@ -524,4 +524,15 @@ input::-webkit-inner-spin-button {
 	.horizontal-scroll-container>* {
 		min-width: 100vw;
 		scroll-snap-align: start;
-	}
\ No newline at end of file
+	}
+
+.header-logo {
+	max-width: 130px;
+	max-height: 130px;
```
Comment (taimur38) [inline web/src/components/Layout/style.css]: test this with not-square logos, vertical rectangle and horizontal rectangle

File: web/src/components/Layout/style.css [hunk 3]
```diff
@@ -524,4 +524,15 @@ input::-webkit-inner-spin-button {
...
+	margin: 0px 480px -130px 80px;
```
Comment (taimur38) [inline web/src/components/Layout/style.css]: test this with lots of different logo sizes

File: web/src/modules/Settings/style.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+.fileContainer [type=file] {
+    cursor: pointer;
+}
+
+.school.logo {
+    border: 1px solid #ddd;
+    border-radius: 15px;
+    padding: 5px;
+    max-width: 150px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
Comment (taimur38) [inline web/src/modules/Settings/style.css]: tell me what this line does

File: web/src/components/Layout/style.css (hunk 2 above)
Comment (raoaliahmad) [reply web/src/components/Layout/style.css]: done

**Pull request merged**